expressions/NOTICE.txt

```diff
@@ -29,6 +29,9 @@ Apache Software License, Version 2.0
 BSD - Scala License
   Scala Library
 
+Eclipse Public License - v 1.0
+  JUnit
+
```

Comment:
I just have a question: how come we are removing Junit4 but we are inlcuding an extra license? Was the new junit dependency being evicted by the version 4.12?

Reply:
You're right, in my head that was being removed, not added. It seems wrong to me. 

Junit is now a transitive dependency of `junit-vintage-engine` getting pulled into the main scope, instead of test scope where it was before because of the direct dependency. But I don't understand why that isn't test scope. Will try making it test scope and then hopefully it goes away.

Reply:
@ncordon that had a bigger impact than I expected so you might want to take another look. I think it's okay now though.